bindings/matrix-sdk-ffi/Cargo.toml

```diff
@@ -21,7 +29,8 @@ anyhow = "1.0.51"
 extension-trait = "1.0.1"
 futures-core = "0.3.17"
 futures-util = { version = "0.3.17", default-features = false }
-matrix-sdk = { path = "../../crates/matrix-sdk", features = ["experimental-timeline", "markdown", "socks"] }
+# FIXME: we currently can't feature flag anything in the api.udl, therefore we must enforce sliding-sync being exposed here..
```

Comment:
So why is there a `sliding-sync` in `[features]` above?

Reply:
because I did the feature-section first and all was fine and dandy until clippy runs with `--no-default-features` and all breaks ... and I unbroke it in a sad way ...

I was actually wondering if that might be something we can mitigate now with the macro-based uniffi? Could we? I'd very much prefer this not pulling in `sliding-sync`-stuff unless the user actively sets the feature-flag. Otherwise I'll have to move the sections...

Reply:
Yes, the proc-macro frontend will help with this, just not in its current state. But I think this is reason enough for me to spend at least some time on it this week.